<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteImport</class>
 <widget class="QWidget" name="PaletteImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imported Palette</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Palette</string>
   </property>
  </widget>
  <widget class="QFrame" name="curFrame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>260</width>
     <height>260</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>260</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QListWidget" name="importFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>276</width>
     <height>260</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>-1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="movement">
    <enum>QListView::Free</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
   <property name="gridSize">
    <size>
     <width>17</width>
     <height>16</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="restorePal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>29</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Active Bank:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="bank2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 2</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 7</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 6</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 5</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 4</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 3</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 1</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="bank8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the active bank to bank 8</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaletteImport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaletteImport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
